### DISCUSSION TITLE
On the madmom version of drum transcription

### DISCUSSION THREAD

**Discussion by nicolaus625** [CPJKU/madmom#519] [General]
There was an automatic drum transcription [model](https://arxiv.org/pdf/1806.06676.pdf) that included in madmom, mentioned at [here](http://ifs.tuwien.ac.at/~vogl/dafx2018/#:~:text=Trained%20models%20are%20available,in%20help%20and%20documentation.). But I cannot find the 0.16.dev version on the PyPI page, nor in the documentation. Could you please provide further instruction on how to use the __madmom/bin/DrumTranscripto__ ?

**Comment by superbock** [comment]
There are no PyPI packages for development versions. You have to install from source. Unfortunately I do not have dedicated instructions on how to use it, but please feel free to post any questions as they arise after having installed it.

**Comment by carrus2049** [comment]
https://github.com/MZehren/Automix
This repo has an instruction for installing it from source. I managed to make it work and getting pretty good transcription result. the computation time is rather long tho(maybe caused by the esembled network.
Good luck on that!
